backend/db/full_data_refresh.py

```diff
@@ -0,0 +1,41 @@
+"""
+Fully refresh the database with the latest data from the enclave API.
+  here's what the run file is doing, i think, for datasets:
+    ./venv/bin/python enclave_wrangler/datasets.py -f -o termhub-csets/datasets/downloads
+"""
+import dateutil.parser
+from datetime import datetime
+
+from backend.db.config import CONFIG
+from backend.db.load import load
+from backend.db.utils import get_db_connection, run_sql
+from enclave_wrangler.datasets import download_favorite_datasets
+from enclave_wrangler.objects_api import download_favorite_objects
+
+
+def refresh_db():
```

Comment:
@Sigfried Can you look at this and give me your thoughts right now? Look at the TODO's and the questions I'm asking. Do they make sense? Does how I'm doing this make sense to you?